test/hotspot/jtreg/runtime/MirrorFrame/Test8003720.java

```diff
@@ -27,7 +27,7 @@
  * @summary Method in interpreter stack frame can be deallocated
  * @library /testlibrary/asm
  * @modules java.base/jdk.internal.misc
- * @compile -XDignore.symbol.file Victim.java
+ * @compile -XDignore.symbol.file -source 21 -target 21 Victim.java
```

Comment:
Why is this necessary?

Reply:
The Asmator uses objectweb ASM to transform Victim, which means ASM needs to parse the javac output Victim.class. If this class wishes to be backported in the future, this flag ensures the output class file is of a constant version that ASM on older versions can consistently parse.